charts/signoz/values.yaml

```diff
@@ -1691,13 +1691,18 @@ otelCollector:
           processors: [logstransform/internal, batch]
           exporters: [clickhouselogsexporter]
 
+  # -- OpAMP config for OtelCollector
+  # @default -- See `values.yaml` for defaults
+  opampConfig:
+    server_endpoint: ws://query-service:4320/v1/opamp
```

Comment:
I don't think this will work.

You will have to use `{{ include "queryService.fullname" . }}` in the configmap instead.